<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>662</width>
    <height>584</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>662</width>
    <height>584</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/vaga.png</normaloff>../../Imagens/vaga.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="CameraButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resource.qrc">
          <normaloff>:/Icones/Icones/Webcam.png</normaloff>:/Icones/Icones/Webcam.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AdicionarButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resource.qrc">
          <normaloff>:/Icones/Icones/aplicativos_0847_Add.png</normaloff>:/Icones/Icones/aplicativos_0847_Add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="label_frame" name="labelFrame">
      <property name="minimumSize">
       <size>
        <width>644</width>
        <height>484</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>644</width>
        <height>484</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>&lt;Backspace para iniciar a camera&gt; </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="menu_Debug"/>
  </widget>
  <action name="actionDebug">
   <property name="text">
    <string>&amp;Debug</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>label_frame</class>
   <extends>QLabel</extends>
   <header location="global">label_frame.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
